ECFR-XML-User-Guide.md

```diff
@@ -2277,8 +2277,8 @@ This is a Table Row element.  It is a sub-element of `<TABLE>`.
 This element is used to create cross references. It has both an identifier and a reference identifier.
 
 ```
-<DIV8 N="� 1.36" NODE="37:1.0.1.1.1.2.69.5" TYPE="SECTION">
-  <HEAD>� 1.36   Revocation of power of attorney; withdrawal of patent attorney or agent.</HEAD>
+<DIV8 N="§ 1.36" NODE="37:1.0.1.1.1.2.69.5" TYPE="SECTION">
+  <HEAD>§ 1.36   Revocation of power of attorney; withdrawal of patent attorney or agent.</HEAD>

```

Comment:
+  <HEAD>§ 1.36   Revocation of power of attorney; withdrawal of patent attorney or agent.</HEAD>